app/js/components/Artifacts.jsx

```diff
@@ -58,12 +52,13 @@ const Artifacts = ({ artifacts, dispatchDeleteArtifact }) => {
     );
 };
 
+Artifacts.contextTypes = {
```

Comment:
Looks like this is a leftover from [c6c87a0](https://github.com/qiime2/qiime-studio-frontend/pull/11/commits/c6c87a0e972d03c5c6deca2038a93be99cc28ea7)